src/app/data/poe-ninja/service/item-overview-http.service.ts

```diff
@@ -30,6 +31,7 @@ export enum ItemOverviewType {
 }
 
 const PATH_TYPE_MAP = {
+  [ItemOverviewType.DeliriumOrb]: 'seeds',
```

Comment:
```suggestion
  [ItemOverviewType.Seed]: 'seeds',
```

Reply:
Fixed in 8223d3cddcded5bdad83b7da5a9aedbecc558edd